### PR TITLE
Add PodioTagItemField type

### PR DESCRIPTION
**Pull request by k1rked**
Resolving this issue: https://github.com/podio/podio-php/issues/130

### CODE REVIEW

**Comment by daniel-sc** [comment]
Hi @k1rked , @rafaelmb ,

I'm not sure if I miss something, but I have not seen any documentation of a 'tag' field - can you give me some hint/link?
In case this is about the general 'tag's (https://developers.podio.com/doc/tags) it should probably not be modeled as a field but rather as an item attribute?

Best, Daniel

**Comment by k1rked** [comment]
I haven't worked with the Podio API in a long time, so I won't be much help, sorry. Seems like the issue was just with Contact type apps, not Standard apps though.

Capture

File: models/PodioItemField.php [hunk 1]
```diff
@@ -1221,3 +1224,14 @@ public function api_friendly_values() {
   }
 
 }
+
+/**
+  * Tag Field
```
Comment (daniel-sc) [inline models/PodioItemField.php]:
```suggestion
  * Tag Field. Undocumented by Podio, but occurs as unremovable default field "Organization" in contact apps.
```

File: models/PodioItemField.php [hunk 2]
```diff
@@ -1221,3 +1224,14 @@ public function api_friendly_values() {
...
+  */
+class PodioTagItemField extends PodioItemField{
```
Comment (daniel-sc) [inline models/PodioItemField.php]:
```suggestion
class PodioTagItemField extends PodioItemField {
```

**Pull request merged**